where-to-travel/src/main/webapp/script.js

```diff
@@ -376,75 +383,134 @@ function populatePlaces(placeArray) {
     showModal(NO_PLACES_HTML_PATH);
   }
   for(let place of placeArray) {
-    // marker creation
-    let placeMarker = new google.maps.Marker({
-      position: place.geometry.location,
-      map: map,
-      title: place.name,
-      icon: PIN_PATH,
-    });
+      console.log(place);
+    if (savedPlacesSet.has(place.place_id) && document.getElementById(place.place_id)) {
```

Comment:
Might be nice to add a comment explaining what this if-else statement is doing

Reply:
Good idea. will do